service/model/release.go

```diff
@@ -109,11 +109,10 @@ func NewReleaseNotification(p Project, r Release) ReleaseNotification {
 }
 
 type GithubRelease struct {
-	ID          int64 // GitHub release ID
-	Name        string
-	Body        string
-	TagName     string  // Git tag associated with the release
-	HTMLURL     url.URL // URL to the release page on GitHub
-	CreatedAt   time.Time
-	PublishedAt time.Time
+	GitTagName string
+	// URL to the release page on GitHub.
+	HTMLURL   url.URL
```

Comment:
`HTMLURL` name feels a bit too clunky..
Is it because `URL` is occupied? In this case, we can simply alias `url` package by smth like `urlx`.

Reply:
We can use `URL`. I was just following github naming but you are right it can be simpler.